zkevm-circuits/src/table.rs

```diff
@@ -177,6 +175,14 @@ pub enum TxFieldTag {
     TxHash,
     /// TxType: Type of the transaction
     TxType,
+    /// Access list address count (EIP-2930)
+    AccessListAddressesLen,
+    /// Access list all storage key count (EIP-2930)
+    AccessListStorageKeysLen,
+    /// Gas cost for access list (EIP-2930)
+    AccessListGasCost,
```

Comment:
We can actually remove this field `AccessListGasCost` and simply use the two above fields, i.e. `AccessListAddressesLen` and `AccessListStorageKeysLen` to calculate the gas cost in the `TxEip2930Gadget`.

Reply:
Removed in commit [6880378](https://github.com/scroll-tech/zkevm-circuits/pull/1045/commits/68803783b1447b8fefe2c1fabd626beae51b8a83).